{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "from function_litref import *"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "articles = get_files_in_folder(\"/home/jules/Documents/phd/obsidian_vault/Article_note/\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'total': 2,\n",
       " 'offset': 0,\n",
       " 'data': [{'paperId': '722c73101a36d6307a42471ed6ac05458ec87029',\n",
       "   'title': 'Comprehensive evaluation of deconvolution methods for human brain gene expression'},\n",
       "  {'paperId': '72fdf28c1adbab503365a698327d595fa96e83c4',\n",
       "   'title': 'Comprehensive evaluation of deconvolution methods for human brain gene expression'}]}"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "title = 'Comprehensive evaluation of deconvolution methods for human brain gene expression.md'\n",
    "\n",
    "\n",
    "test = search_by_title(title=title)\n",
    "test\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'paperId': 'ab82bdfb1710a470d4358afa312d7c9da531632b',\n",
       " 'title': 'Guidance for RNA-seq co-expression network construction and analysis: safety in numbers',\n",
       " 'references': [{'paperId': '5e814fc3615ed7e9a55b33510998e91ceca7672a',\n",
       "   'title': 'Measuring the wisdom of the crowds in network-based gene function inference'},\n",
       "  {'paperId': '0446839238dc22746e99eb746ce11bc7e18f33b1',\n",
       "   'title': 'A comprehensive assessment of RNA-seq accuracy, reproducibility and information content by the Sequencing Quality Control consortium'},\n",
       "  {'paperId': 'ac5e0caf4ff09abff6e4a51c1e1388dae7f7eaf8',\n",
       "   'title': 'Detecting and correcting systematic variation in large-scale RNA sequencing data'},\n",
       "  {'paperId': '17868f976dd014be13acaf1ceff94c802abf14a0',\n",
       "   'title': 'RNA-seq differential expression studies: more sequence or more replication?'},\n",
       "  {'paperId': '0fa6c95598393f7fa0c9969d6c3856b635c25470',\n",
       "   'title': 'Integrative Functional Genomic Analyses Implicate Specific Molecular Pathways and Circuits in Autism'},\n",
       "  {'paperId': 'd6720c598d391cc4f2cf6d6ac0b1a779070e6804',\n",
       "   'title': 'Maize Gene Atlas Developed by RNA Sequencing and Comparative Evaluation of Transcriptomes Based on RNA Sequencing and Microarrays'},\n",
       "  {'paperId': '9c05cc4538a272adc0006a2d3e1967b34ce6d1c8',\n",
       "   'title': 'Genes, behavior and next‚Äêgeneration RNA sequencing'},\n",
       "  {'paperId': 'd244ba9e48a3afd4e67935a98a2d49272911f6ee',\n",
       "   'title': 'Gemma: a resource for the reuse, sharing and meta-analysis of expression profiling data'},\n",
       "  {'paperId': '5dcb131e1b335b14f712e31bc6a487e697f1b29b',\n",
       "   'title': 'GENCODE: The reference human genome annotation for The ENCODE Project'},\n",
       "  {'paperId': '512e5a0dc0ba7568b7364ed54781aa88d1cd693d',\n",
       "   'title': 'Utilizing RNA-Seq data for de novo coexpression network inference'},\n",
       "  {'paperId': '6fca260e9a3c37e246cc15e7639d1a5fa2aed465',\n",
       "   'title': 'Fast gapped-read alignment with Bowtie 2'},\n",
       "  {'paperId': 'f540e3d5268ca64097425e08d6059de458f558dc',\n",
       "   'title': 'ReCount: A multi-experiment resource of analysis-ready RNA-seq gene count datasets'},\n",
       "  {'paperId': 'd50458c350897b517ac409a568b941e13f071e76',\n",
       "   'title': 'RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome'},\n",
       "  {'paperId': '0aa840ef8613f7ad2ca6597e4be4a2ccb8ed02f5',\n",
       "   'title': 'The role of indirect connections in gene networks in predicting function'},\n",
       "  {'paperId': '4f183ba71be8873857aebb2199d2a1fdc75adcd3',\n",
       "   'title': 'parmigene - a parallel R package for mutual information estimation and gene network reconstruction'},\n",
       "  {'paperId': '9ddba1f5f559341fb713fd7445028cfdd40ed09b',\n",
       "   'title': 'Transcriptomic Analysis of Autistic Brain Reveals Convergent Molecular Pathology'},\n",
       "  {'paperId': '7cdead5f2ad8a51972ba3ba992a61168e8e11650',\n",
       "   'title': 'The Impact of Multifunctional Genes on \"Guilt by Association\" Analysis'},\n",
       "  {'paperId': '83b2e0d77f3d7c4a2f296856fb09a68a1e097f13',\n",
       "   'title': 'The Sequence Read Archive'},\n",
       "  {'paperId': 'f8dc73630864b4903f8f57b5c8d7b098eb300e4f',\n",
       "   'title': 'Reactome: a database of reactions, pathways and biological processes'},\n",
       "  {'paperId': 'a834b996c8ba92ea55c6dd7f4a710de6d4b4dc7c',\n",
       "   'title': 'Statistical Design and Analysis of RNA Sequencing Data'},\n",
       "  {'paperId': '73e87460a00939120ef7be676f4cb65de7e08212',\n",
       "   'title': 'A global meta-analysis of microarray expression data to predict unknown gene functions and estimate the literature-data divide'},\n",
       "  {'paperId': 'ef117c95b92b68b751143155022a5c1a600afe5c',\n",
       "   'title': 'Mapping and quantifying mammalian transcriptomes by RNA-Seq'},\n",
       "  {'paperId': '0859f4bc8e46b6b1fcfa5a75d5a703b71cb64ea9',\n",
       "   'title': 'GeneMANIA: a real-time multiple association network integration algorithm for predicting gene function'},\n",
       "  {'paperId': '230444cd55ea81d4210dd6980a46cc0fa782c330',\n",
       "   'title': 'A critical assessment of Mus musculus gene function prediction using integrated genomic evidence'},\n",
       "  {'paperId': '268a4f8da15a42f3e0e71691f760ff5edbf9cec8',\n",
       "   'title': 'LIBLINEAR: A Library for Large Linear Classification'},\n",
       "  {'paperId': 'c6480aecd32cff4ebbd9d12e0c351249cd76a760',\n",
       "   'title': 'GEOquery: a bridge between the Gene Expression Omnibus (GEO) and BioConductor'},\n",
       "  {'paperId': '1162bb9da2c7cf2d92c2e119eb3aa468fe5825ef',\n",
       "   'title': 'Large-Scale Mapping and Validation of Escherichia coli Transcriptional Regulation from a Compendium of Expression Profiles'},\n",
       "  {'paperId': '00ae5e12df354c4c70b7d51863c6f9f15e44a46a',\n",
       "   'title': 'A General Framework for Weighted Gene Co-Expression Network Analysis'},\n",
       "  {'paperId': 'e8ed956ce6a454ffb74a915f02280799eb1f133a',\n",
       "   'title': 'ARACNE: An Algorithm for the Reconstruction of Gene Regulatory Networks in a Mammalian Cellular Context'},\n",
       "  {'paperId': '0ef7d9e618cbb507d69f8ebcdc60b8a1f3135bff',\n",
       "   'title': 'Solving large scale linear prediction problems using stochastic gradient descent algorithms'},\n",
       "  {'paperId': 'ef8161b95ba86b2644b56bfffa54f0926843376a',\n",
       "   'title': 'Coexpression analysis of human genes across many microarray data sets.'},\n",
       "  {'paperId': 'e7b9bb433eb30eae9bbec7d4ae97903a6718c083',\n",
       "   'title': 'Linear Models and Empirical Bayes Methods for Assessing Differential Expression in Microarray Experiments'},\n",
       "  {'paperId': '3ca4d4229b8a843c0847fc70531790df6bd017ec',\n",
       "   'title': 'Online Passive-Aggressive Algorithms'},\n",
       "  {'paperId': 'ba969a4f3bc5f5a84f6025478e566c40661d85f3',\n",
       "   'title': 'Feature selection based on mutual information criteria of max-dependency, max-relevance, and min-redundancy'},\n",
       "  {'paperId': 'e6c4925fb114d13a8568f88957c167c928f0c9f1',\n",
       "   'title': 'The Structure and Function of Complex Networks'},\n",
       "  {'paperId': 'a411f6a0e6473137ac1a538f7cee65722fa3584f',\n",
       "   'title': 'Gene Ontology: tool for the unification of biology'},\n",
       "  {'paperId': '70b6cffd868290b93c71bdd1b6c81cf02d78fd9b',\n",
       "   'title': 'Proteomics: Guilt-by-association goes global'},\n",
       "  {'paperId': '8573bbb55112c1f06c43211daedacc1578e69a92',\n",
       "   'title': 'Cluster analysis and display of genome-wide expression patterns.'},\n",
       "  {'paperId': None,\n",
       "   'title': 'Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation'},\n",
       "  {'paperId': '8e0be569ea77b8cb29bb0e8b031887630fe7a96c',\n",
       "   'title': 'Random Forests'},\n",
       "  {'paperId': '55557c0ebc71ff48881ef0c8531fc1bea4c28b5c',\n",
       "   'title': 'KEGG: Kyoto Encyclopedia of Genes and Genomes'},\n",
       "  {'paperId': '0daeb998ecbc127ef164ae375618013002d176a7',\n",
       "   'title': 'Bioinformatics Applications Note Gene Expression Rseqtools: a Modular Framework to Analyze Rna-seq Data Using Compact, Anonymized Data Summaries'},\n",
       "  {'paperId': None, 'title': 'Pattern Anal. Mach. Intell'},\n",
       "  {'paperId': None,\n",
       "   'title': 'BrainSpan: Atlas of the Developing Human Brain'}]}"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "infos = get_paper_info(\"ab82bdfb1710a470d4358afa312d7c9da531632b\")\n",
    "infos"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'paperId': '5e814fc3615ed7e9a55b33510998e91ceca7672a',\n",
       "  'title': 'Measuring the wisdom of the crowds in network-based gene function inference'},\n",
       " {'paperId': '0446839238dc22746e99eb746ce11bc7e18f33b1',\n",
       "  'title': 'A comprehensive assessment of RNA-seq accuracy, reproducibility and information content by the Sequencing Quality Control consortium'},\n",
       " {'paperId': 'ac5e0caf4ff09abff6e4a51c1e1388dae7f7eaf8',\n",
       "  'title': 'Detecting and correcting systematic variation in large-scale RNA sequencing data'},\n",
       " {'paperId': '17868f976dd014be13acaf1ceff94c802abf14a0',\n",
       "  'title': 'RNA-seq differential expression studies: more sequence or more replication?'},\n",
       " {'paperId': '0fa6c95598393f7fa0c9969d6c3856b635c25470',\n",
       "  'title': 'Integrative Functional Genomic Analyses Implicate Specific Molecular Pathways and Circuits in Autism'},\n",
       " {'paperId': 'd6720c598d391cc4f2cf6d6ac0b1a779070e6804',\n",
       "  'title': 'Maize Gene Atlas Developed by RNA Sequencing and Comparative Evaluation of Transcriptomes Based on RNA Sequencing and Microarrays'},\n",
       " {'paperId': '9c05cc4538a272adc0006a2d3e1967b34ce6d1c8',\n",
       "  'title': 'Genes, behavior and next‚Äêgeneration RNA sequencing'},\n",
       " {'paperId': 'd244ba9e48a3afd4e67935a98a2d49272911f6ee',\n",
       "  'title': 'Gemma: a resource for the reuse, sharing and meta-analysis of expression profiling data'},\n",
       " {'paperId': '5dcb131e1b335b14f712e31bc6a487e697f1b29b',\n",
       "  'title': 'GENCODE: The reference human genome annotation for The ENCODE Project'},\n",
       " {'paperId': '512e5a0dc0ba7568b7364ed54781aa88d1cd693d',\n",
       "  'title': 'Utilizing RNA-Seq data for de novo coexpression network inference'},\n",
       " {'paperId': '6fca260e9a3c37e246cc15e7639d1a5fa2aed465',\n",
       "  'title': 'Fast gapped-read alignment with Bowtie 2'},\n",
       " {'paperId': 'f540e3d5268ca64097425e08d6059de458f558dc',\n",
       "  'title': 'ReCount: A multi-experiment resource of analysis-ready RNA-seq gene count datasets'},\n",
       " {'paperId': 'd50458c350897b517ac409a568b941e13f071e76',\n",
       "  'title': 'RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome'},\n",
       " {'paperId': '0aa840ef8613f7ad2ca6597e4be4a2ccb8ed02f5',\n",
       "  'title': 'The role of indirect connections in gene networks in predicting function'},\n",
       " {'paperId': '4f183ba71be8873857aebb2199d2a1fdc75adcd3',\n",
       "  'title': 'parmigene - a parallel R package for mutual information estimation and gene network reconstruction'},\n",
       " {'paperId': '9ddba1f5f559341fb713fd7445028cfdd40ed09b',\n",
       "  'title': 'Transcriptomic Analysis of Autistic Brain Reveals Convergent Molecular Pathology'},\n",
       " {'paperId': '7cdead5f2ad8a51972ba3ba992a61168e8e11650',\n",
       "  'title': 'The Impact of Multifunctional Genes on \"Guilt by Association\" Analysis'},\n",
       " {'paperId': '83b2e0d77f3d7c4a2f296856fb09a68a1e097f13',\n",
       "  'title': 'The Sequence Read Archive'},\n",
       " {'paperId': 'f8dc73630864b4903f8f57b5c8d7b098eb300e4f',\n",
       "  'title': 'Reactome: a database of reactions, pathways and biological processes'},\n",
       " {'paperId': 'a834b996c8ba92ea55c6dd7f4a710de6d4b4dc7c',\n",
       "  'title': 'Statistical Design and Analysis of RNA Sequencing Data'},\n",
       " {'paperId': '73e87460a00939120ef7be676f4cb65de7e08212',\n",
       "  'title': 'A global meta-analysis of microarray expression data to predict unknown gene functions and estimate the literature-data divide'},\n",
       " {'paperId': 'ef117c95b92b68b751143155022a5c1a600afe5c',\n",
       "  'title': 'Mapping and quantifying mammalian transcriptomes by RNA-Seq'},\n",
       " {'paperId': '0859f4bc8e46b6b1fcfa5a75d5a703b71cb64ea9',\n",
       "  'title': 'GeneMANIA: a real-time multiple association network integration algorithm for predicting gene function'},\n",
       " {'paperId': '230444cd55ea81d4210dd6980a46cc0fa782c330',\n",
       "  'title': 'A critical assessment of Mus musculus gene function prediction using integrated genomic evidence'},\n",
       " {'paperId': '268a4f8da15a42f3e0e71691f760ff5edbf9cec8',\n",
       "  'title': 'LIBLINEAR: A Library for Large Linear Classification'},\n",
       " {'paperId': 'c6480aecd32cff4ebbd9d12e0c351249cd76a760',\n",
       "  'title': 'GEOquery: a bridge between the Gene Expression Omnibus (GEO) and BioConductor'},\n",
       " {'paperId': '1162bb9da2c7cf2d92c2e119eb3aa468fe5825ef',\n",
       "  'title': 'Large-Scale Mapping and Validation of Escherichia coli Transcriptional Regulation from a Compendium of Expression Profiles'},\n",
       " {'paperId': '00ae5e12df354c4c70b7d51863c6f9f15e44a46a',\n",
       "  'title': 'A General Framework for Weighted Gene Co-Expression Network Analysis'},\n",
       " {'paperId': 'e8ed956ce6a454ffb74a915f02280799eb1f133a',\n",
       "  'title': 'ARACNE: An Algorithm for the Reconstruction of Gene Regulatory Networks in a Mammalian Cellular Context'},\n",
       " {'paperId': '0ef7d9e618cbb507d69f8ebcdc60b8a1f3135bff',\n",
       "  'title': 'Solving large scale linear prediction problems using stochastic gradient descent algorithms'},\n",
       " {'paperId': 'ef8161b95ba86b2644b56bfffa54f0926843376a',\n",
       "  'title': 'Coexpression analysis of human genes across many microarray data sets.'},\n",
       " {'paperId': 'e7b9bb433eb30eae9bbec7d4ae97903a6718c083',\n",
       "  'title': 'Linear Models and Empirical Bayes Methods for Assessing Differential Expression in Microarray Experiments'},\n",
       " {'paperId': '3ca4d4229b8a843c0847fc70531790df6bd017ec',\n",
       "  'title': 'Online Passive-Aggressive Algorithms'},\n",
       " {'paperId': 'ba969a4f3bc5f5a84f6025478e566c40661d85f3',\n",
       "  'title': 'Feature selection based on mutual information criteria of max-dependency, max-relevance, and min-redundancy'},\n",
       " {'paperId': 'e6c4925fb114d13a8568f88957c167c928f0c9f1',\n",
       "  'title': 'The Structure and Function of Complex Networks'},\n",
       " {'paperId': 'a411f6a0e6473137ac1a538f7cee65722fa3584f',\n",
       "  'title': 'Gene Ontology: tool for the unification of biology'},\n",
       " {'paperId': '70b6cffd868290b93c71bdd1b6c81cf02d78fd9b',\n",
       "  'title': 'Proteomics: Guilt-by-association goes global'},\n",
       " {'paperId': '8573bbb55112c1f06c43211daedacc1578e69a92',\n",
       "  'title': 'Cluster analysis and display of genome-wide expression patterns.'},\n",
       " {'paperId': None,\n",
       "  'title': 'Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation'},\n",
       " {'paperId': '8e0be569ea77b8cb29bb0e8b031887630fe7a96c',\n",
       "  'title': 'Random Forests'},\n",
       " {'paperId': '55557c0ebc71ff48881ef0c8531fc1bea4c28b5c',\n",
       "  'title': 'KEGG: Kyoto Encyclopedia of Genes and Genomes'},\n",
       " {'paperId': '0daeb998ecbc127ef164ae375618013002d176a7',\n",
       "  'title': 'Bioinformatics Applications Note Gene Expression Rseqtools: a Modular Framework to Analyze Rna-seq Data Using Compact, Anonymized Data Summaries'},\n",
       " {'paperId': None, 'title': 'Pattern Anal. Mach. Intell'},\n",
       " {'paperId': None, 'title': 'BrainSpan: Atlas of the Developing Human Brain'}]"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "infos[\"references\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['A benchmark for RNA-seq deconvolution analysis under dynamic testing environments',\n",
       " 'Genome-wide association study implicates novel loci and reveals candidate effector genes for longitudinal pediatric bone accrual',\n",
       " 'Benchmarking of cell type deconvolution pipelines for transcriptomics data',\n",
       " 'Single-nucleus transcriptomics of the prefrontal cortex in major depressive disorder implicates oligodendrocyte precursor cells and excitatory neurons',\n",
       " 'SINGLE',\n",
       " 'Correction: Differential transcriptional response following glucocorticoid activation in cultured blood immune cells: a novel approach to PTSD biomarker development',\n",
       " 'Single-cell reconstruction of the adult human heart during heart failure and recovery reveals the cellular landscape underlying cardiac function',\n",
       " 'deconvSeq: deconvolution of cell mixture distribution in sequencing data',\n",
       " 'The Role of Scale in the Estimation of Cell-type Proportions',\n",
       " 'The GTEx Consortium atlas of genetic regulatory effects across human tissues',\n",
       " 'CommonMind Consortium provides transcriptomic and epigenomic data for Schizophrenia and Bipolar Disorder',\n",
       " 'TOAST: improving reference-free cell composition estimation by cross-cell type differential analysis',\n",
       " 'A community-based transcriptomics classification and nomenclature of neocortical cell types',\n",
       " 'Conserved cell types with divergent features in human versus mouse cortex',\n",
       " 'Using multiple measurements of tissue to estimate subject- and cell-type-specific gene expression',\n",
       " 'Pairwise common variant meta-analyses of schizophrenia with other psychiatric disorders reveals shared and distinct gene and gene-set associations',\n",
       " 'Cell type-specific epigenetic links to schizophrenia risk in the brain',\n",
       " 'Accurate estimation of cell-type composition from gene expression data',\n",
       " 'Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology',\n",
       " 'Comprehensive Integration of Single-Cell Data',\n",
       " 'Complete deconvolution of cellular mixtures based on linearity of transcriptional signatures',\n",
       " 'Single-cell genomics identifies cell type‚Äìspecific molecular changes in autism',\n",
       " 'g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update)',\n",
       " 'Determining cell-type abundance and expression from bulk tissues with digital cytometry',\n",
       " 'Cell composition analysis of bulk genomics using single cell data',\n",
       " 'Identification of common genetic risk variants for autism spectrum disorder',\n",
       " 'Diabetes Relief in Mice by Glucose-Sensing Insulin-Secreting Human Œ±-Cells',\n",
       " 'Comprehensive functional genomic resource and integrative model for the human brain',\n",
       " 'Transcriptomic Landscape of von Economo Neurons in Human Anterior Cingulate Cortex Revealed by Microdissected-Cell RNA Sequencing',\n",
       " 'Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage',\n",
       " 'Leveraging heterogeneity across multiple datasets increases cell-mixture deconvolution accuracy and reduces biological and technical biases',\n",
       " 'Dtangle: Accurate and Robust Cell Type Deconvolution',\n",
       " 'Shared and distinct transcriptomic cell types across neocortical areas',\n",
       " 'Inference of cell type content from human brain transcriptomic datasets illuminates the effects of age, manner of death, dissection, and psychiatric diagnosis',\n",
       " 'LONGO: an R package for interactive gene length dependent analysis for neuronal identity',\n",
       " 'Bulk tissue cell type deconvolution with multi-subject single-cell expression reference',\n",
       " 'Computational deconvolution of transcriptomics data from mixed cell populations',\n",
       " 'Cell-Type Heterogeneity in Adipose Tissue Is Associated with Complex Traits and Reveals Disease-Relevant Cell-Specific eQTLs',\n",
       " 'Genetic variants associated with Alzheimer‚Äôs disease confer different cerebral cortex cell-type population structure',\n",
       " 'Variation among intact tissue samples reveals the core transcriptional features of human CNS cell classes',\n",
       " 'Integrative single-cell analysis of transcriptional and epigenetic states in the human adult brain',\n",
       " 'Genetic effects on gene expression across human tissues',\n",
       " 'Single-Cell Analysis of Human Pancreas Reveals Transcriptional Signatures of Aging and Somatic Mutation Patterns',\n",
       " 'Cross-Laboratory Analysis of Brain Cell Type Transcriptomes with Applications to Interpretation of Bulk Tissue Data',\n",
       " 'Estimation of immune cell content in tumour tissue using single-cell RNA-seq data',\n",
       " 'xCell: digitally portraying the tissue cellular heterogeneity landscape',\n",
       " 'Genome-wide changes in lncRNA, splicing, and regional gene expression patterns in autism',\n",
       " 'Ultrastructural Characterization of the Lower Motor System in a Mouse Model of Krabbe Disease',\n",
       " 'CIDR: Ultrafast and accurate clustering through imputation for single-cell RNA-seq data',\n",
       " 'Post-mortem molecular profiling of three psychiatric disorders',\n",
       " 'Mathematical modelling of transcriptional heterogeneity identifies novel markers and subpopulations in complex tissues',\n",
       " 'Purification and Characterization of Progenitor and Mature Human Astrocytes Reveals Transcriptional and Functional Differences with Mouse',\n",
       " 'A Critical Survey of Deconvolution Methods for Separating Cell Types in Complex Tissues',\n",
       " 'A survey of human brain transcriptome diversity at the single cell level',\n",
       " 'Novel Observations From Next-Generation RNA Sequencing of Highly Purified Human Adult and Fetal Islet Cell Subsets',\n",
       " 'Robust enumeration of cell subsets from tissue expression profiles',\n",
       " 'Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2',\n",
       " 'An RNA-Sequencing Transcriptome and Splicing Database of Glia, Neurons, and Vascular Cells of the Cerebral Cortex',\n",
       " 'Convex Analysis of Mixtures for Separating Non-negative Well-grounded Sources',\n",
       " 'Transcriptional Landscape of the Prenatal Human Brain',\n",
       " 'A promoter-level mammalian expression atlas',\n",
       " 'Cell type-specific analysis of human brain transcriptome data to predict alterations in cellular composition',\n",
       " 'DeconRNASeq: a statistical framework for deconvolution of heterogeneous tissue samples based on mRNA-Seq data',\n",
       " 'Digital sorting of complex tissues for cell type-specific gene expression profiles',\n",
       " 'A cell epigenotype specific model for the correction of brain cellular heterogeneity bias and its application to age, brain region and major depression',\n",
       " 'PERT: A Method for Expression Deconvolution of Human Blood Samples from Varied Microenvironmental and Developmental Conditions',\n",
       " 'Cell population-specific expression analysis of human cerebellum',\n",
       " 'Landscape of transcription in human cells',\n",
       " 'RBFOX1 regulates both splicing and transcriptional networks in human neuronal development.',\n",
       " 'Gene expression deconvolution in linear space',\n",
       " 'Cell type‚Äìspecific gene expression differences in complex tissues',\n",
       " 'Deconvolution of Blood Microarray Data Identifies Cellular Activation Patterns in Systemic Lupus Erythematosus',\n",
       " 'Neocortical glial cell numbers in human brains',\n",
       " 'Eigengene networks for studying the relationships between co-expression modules',\n",
       " 'Cell-type-specific expression for non-brain tissues were accessed from the following four sources',\n",
       " 'UNDO: a Bioconductor R package for unsupervised deconvolution of mixed gene expressions in tumor samples',\n",
       " 'Gene expression in the human brain: the current state of the study of specificity and spatiotemporal dynamics.',\n",
       " 'STAR: ultrafast universal RNA-seq aligner',\n",
       " 'Visualizing Data using t-SNE',\n",
       " 'Maturation of a central',\n",
       " 'Controlling the false discovery rate: a practical and powerful approach to multiple testing']"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "get_reference_titles(title)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "vault_path = \"/home/jules/Documents/phd/obsidian_vault/Article_note/\"\n",
    "title = \"Comprehensive evaluation of deconvolution methods for human brain gene expression.md\"\n",
    "\n",
    "\n",
    "add_references(title,vault_path)\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
